react-app/src/components/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -9,13 +9,14 @@ import rehypeSanitize from "rehype-sanitize";
 import cn from "classnames";
 
 interface MarkdownEditorProps
-  extends Omit<MDEditorProps, "className" | "onChange"> {
-  className?: string;
-  onChange: (value: string) => void;
+  extends Omit<MDEditorProps, "onChange" | "value"> {
+  containerClassName?: string;
+  value: string;
+  setValue: (value: string) => void;
```

Comment:
are these initialValue & onValueChange?

Reply:
not exactly initialValue since the state is managed by the component comsumer and the value actually reflects the current value? agree on the `onValueChange` tho

Reply:
ok